rust/kcl-lib/src/std/sketch.rs

```diff
@@ -1019,14 +1049,20 @@ pub async fn start_sketch_on(exec_state: &mut ExecState, args: Args) -> Result<K
 #[stdlib {
     name = "startSketchOn",
     feature_tree_operation = true,
+    keywords = true,
+    unlabeled_first = true,
+    args = {
+        plane_or_solid = { docs = "The plane or solid to sketch on"},
+        tag = { docs = "Identify a face of a solid if "},
```

Comment:
Wouldn't a better name be "face"? The arg doesn't need to be a tag, like with end caps. I also thought the convention could be that a tag parameter always creates a tag.

Also, doc seems to be cut off.

Reply:
OK, that makes sense (although I think of 'start'/'end' as kind of generic tags)

Reply:
Hmm, I see what you mean. I guess either way could work.